src/main/java/study/lotto/domain/LottoNumber.java

```diff
@@ -41,4 +55,9 @@ public int hashCode() {
     public String toString() {
         return String.valueOf(number);
     }
+
+    @Override
+    public int compareTo(LottoNumber o) {
+        return this.number - o.number;
+    }
```

Comment:
compreTo 구현시에 `-` 연산을 사용하면 오버플로가 발생하거나 실수비교시에는 부동소수점 계산 오류가 발생할 수 있다고해요.
아래와 같은 방법을 권해드립니다. 😁 
```suggestion
    @Override
    public int compareTo(LottoNumber o) {
        return Integer.compare(number, o.number);
    }
```

Reply:
감사합니다. !!
몰랐으면 찾기 힘든 부분인 것 같습니다. 
큰일날 뻔 했습니다.
감사합니다 !!